connectors/meilisearch/index.js

```diff
@@ -76,6 +76,8 @@ module.exports = async ({ storeConnector, collectionConnector }) => {
      */
     waitForTask: async function ({ collection, taskUid }) {
       try {
+        console.log({ collection, taskUid })
```

Comment:
👀

Reply:
The whole function should not even be there! Thanks